src/main/java/com/google/devtools/build/lib/rules/java/JavaOptions.java

```diff
@@ -612,31 +612,31 @@ public ImportDepsCheckingLevelConverter() {
 
   @Option(
       name = "java_runtime_version",
-      defaultValue = "",
+      defaultValue = "local_jdk",
       documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
       effectTags = {OptionEffectTag.UNKNOWN},
       help = "The Java runtime version")
   public String javaRuntimeVersion;
 
   @Option(
       name = "tool_java_runtime_version",
-      defaultValue = "",
+      defaultValue = "remotejdk_11",
       documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
       effectTags = {OptionEffectTag.UNKNOWN},
       help = "The Java runtime version used to execute tools during the build")
   public String hostJavaRuntimeVersion;
 
   @Option(
       name = "java_language_version",
-      defaultValue = "",
+      defaultValue = "8",
```

Comment:
Note: The plan is to update this to "11" when the machinery is a bit better and it will work on Java 8.
For this change not to cause problems internally it looks best option is to override it in .rc file.

Reply:
SGTM